site/content@beta/backend-apis/apis.md

```diff
@@ -0,0 +1,47 @@
+---
+weight: 40
+---
+
+# Agent Chat API
+
+## Use cases
+
+The [**Agent Chat API**](../agent-chat-api) allows for: 
+
+- interaction with the chat as Agent (joining a chat, posting messages)
+- interact with the chat by Bot Agents (acting as Agents) 
+- building a custom Agent App
+- browsing chat archives
+- banning Customers
+
+## Chatting as Agent
+
+The number of chats an Agent can participate in is not limited. It can be configured in the Agent App by the licence Owner or Admin.
+
+# Customer Chat API
+
+## Use cases
+
+The [**Customer Chat API**](../customer-chat-api) allows for:
+
+- interacting with the chat as Customer (join a chat, post messages)
+- building a custom Chat Widget
+
+## Chatting as Customer
+
+By default, a Customer can only have one chat started, unless he chats with Agents who use different licences. The licence Owner or Admin can change that default value, modifying the `routing.max_customer_chats_count` property (link to REST API needed). However, in order to support this functionality, you would need to build a custom Chat Widget. For now, the LiveChat Chat Widget doesn't support it. You can build a Chat Widget from the ground up using the [Chat Widget Customer SDK](../customer-sdk/).
```

Comment:
"For now, the LiveChat Chat Widget doesn’t support it." -> "For now, the default LiveChat Chat Widget doesn’t support it.

Reply:
tego mi brakowało w tym zdaniu :P